src/semidiscretization/semidiscretization.jl

```diff
@@ -15,6 +15,19 @@ Return the number of degrees of freedom associated with each scalar variable.
     ndofs(mesh, solver, cache)
 end
 
+"""
+    ndofsglobal(semi::AbstractSemidiscretization)
+
+Return the global number of degrees of freedom associated with each scalar variable.
```

Comment:
```suggestion
Return the global number of degrees of freedom associated with each scalar variable across all MPI ranks.
```
Just to clarify what is meant by "global"